ntc_templates/templates/aruba_aoscx_show_vsf_detail.textfsm

```diff
@@ -1,18 +1,24 @@
 Value MEMBER_ID (\S+)
+Value TOPOLOGY (\S+)
+Value NAME (\S+)
 Value MAC_ADDRESS (\S+)
-Value STATUS (.*)
+Value STATUS (\S+.*)
+Value SERIAL_NUMBER (\S+)
 Value CPU (\d+%)
 Value MEMORY (\d+%)
 Value VSF_LINK_1 (.*)
 Value VSF_LINK_2 (.*)
 
 Start
-  ^VSF\s*${MEMBER_ID}
+  ^VSF\s+${MEMBER_ID}
+  ^\s+Topology\s+:\s+${TOPOLOGY}
+  ^\s+Name\s+:\s+${NAME}
   ^Member.* -> Continue.Record
   ^Member\s+ID\s+:\s+${MEMBER_ID}
-  ^\s+MAC\s+Address\s+:\s+${MAC_ADDRESS}                                                                          
-  ^\s+Status\s+:\s+${STATUS}                                                                       
-  ^\s+CPU\sUtilization\s+:\s+${CPU}                                                                      
+  ^\s+MAC\s+Address\s+:\s+${MAC_ADDRESS}
+  ^\s+Status\s+:\s+${STATUS}
+  ^\s+Serial\sNumber\s+:\s+${SERIAL_NUMBER}
+  ^\s+CPU\sUtilization\s+:\s+${CPU}
```

Comment:
While there isn't more than one space today, adding the plus sign future proofs us (at the cost of an extra character).

```suggestion
  ^\s+Serial\s+Number\s+:\s+${SERIAL_NUMBER}
  ^\s+CPU\s+Utilization\s+:\s+${CPU}
```

Reply:
Fair, thanks.